js/message_test.js

```diff
@@ -184,14 +180,7 @@ describe('Message test suite', function() {
     var response = new proto.jspb.test.DefaultValues();
 
     // Test toObject
-    var expectedObject = {
-      stringField: defaultString,
-      boolField: true,
-      intField: 11,
-      enumField: 13,
-      emptyField: '',
-      bytesField: 'bW9v'
-    };
+    var expectedObject = {};
```

Comment:
> Making sure because some assertions will need to be changed either way.

Why? I am suggesting to keep exactly the current semantics, where fields with and without explicit defaults behave differently. That shouldn't require changing any assertions.